csrankings.csv

```diff
@@ -20459,7 +20459,7 @@ Xiaowen Fang,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facu
 Xiaowen Yuan,Massey University,https://www.massey.ac.nz/massey/learning/colleges/college-of-sciences/staff-profile.cfm?stref=737350,n2acINwAAAAJ
 Xiaowen Zhang,CUNY,http://www.csi.cuny.edu/faculty/ZHANG_XIAOWEN.html,vX1p5wIAAAAJ
 Xiaoxing Ma,Nanjing University,http://ics.nju.edu.cn/people/xiaoxingma,44WpWR4AAAAJ
-Xiaoxuan Lu 0001,University of Edinburgh,https://christopherlu.github.io,idu78-EAAAAJ
+Chris Xiaoxuan Lu,University of Edinburgh,https://christopherlu.github.io,idu78-EAAAAJ

```

Comment:
Both entries need to be included and you need to add an entry to `dblp-aliases.csv`.

Reply:
Thanks, Emery. Modification is now made.